src/directives/vRotate.ts

```diff
@@ -0,0 +1,61 @@
+import { shallowRef } from 'vue'
+import { Quaternion, Vector3 } from 'three'
+import type { TresObject } from '../types'
+import { useLogger, useRenderLoop } from '../composables'
+
+const { logWarning } = useLogger()
+
+export const vRotate = {
+  mounted: (
+    el: TresObject,
+    binding: {
+      arg: 'x' | 'y' | 'z'
+      value: number
+      modifiers: Partial<{ x: boolean; y: boolean; z: boolean }>
+    },
+  ) => {
+    if (el.isCamera) {
+      logWarning(`Rotate the ${el.type} is not a good idea`)
+      return
+    }
+    const speed = binding.value ?? 0.01
```

Comment:
(Non-blocking)

I personally find it really helpful if this sort of variable is named using units. Here, that would be `radiansPerFrame` (on a single axis), if I'm understanding the code correctly.

Something like `speed` is fine if I just want to tweak a value until it looks right. But particularly for cases where I want to sync the movement of two different components, or where I might want to include an equation from a physics textbook, it's *really* handy to have the units in the variable name.

Reply:
I like this one. Thanks :)